src/chrome-launcher.ts

```diff
@@ -205,7 +209,7 @@ class Launcher {
       }
     }
     if (this.chromePath === undefined) {
-      const installations = await chromeFinder[getPlatform() as SupportedPlatforms]();
+      const installations = (<typeof Launcher>this.constructor).getInstallations();
```

Comment:
in case it's not obvious, because `Launcher` is publicly exported, rather than reference `Launcher.getInstallations()`, I opted to use late static binding here so that consumers that wish to override `getInstallations()` can.

Reply:
I'm not sure what value that really has. We already provide `chromePath` as an option and we'd never do anything with installations past the first one.

Reply:
that's a super fair and valid point.

I suppose my inclination is to lean towards ensuring that public class methods can be overridden by consumers of the library --but you've made a great point that, currently, the only call-site that uses this already supports being overridden via another interface.

I'll replace with `Launcher.getInstallations()` 😄

Reply:
done!